huggingface/pytorch/inference/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -22,7 +22,8 @@ ARG OPEN_MPI_VERSION=4.1.4
 ARG MMS_VERSION=1.1.8
 
 # HF ARGS
-ARG TORCH_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r1.13.1_ec2/20221217-194030/54406b8eed7fbd61be629cb06229dfb7b6b2954e/torch-1.13.1%2Bcpu-cp39-cp39-linux_x86_64.whl
+ARG TORCH_VERSION=1.13.1
+ARG TORCH_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r${TORCH_VERSION}_ec2/20221217-194030/54406b8eed7fbd61be629cb06229dfb7b6b2954e/torch-${TORCH_VERSION}%2Bcpu-cp39-cp39-linux_x86_64.whl
```

Comment:
Not sure if `${TORCH_VERSION}` is needed since the whole "URI" is dynamic. So if the version changes the whole string needs to be replaced and cannot be derived from the version change, e.g. the date `20221217` is included